package.json

```diff
@@ -154,10 +153,8 @@
 		"@testing-library/vue": "^5.8.2",
 		"app-manifest-loader": "^2.4.1",
 		"autoprefixer": "^10.4.2",
-		"babel-core": "^7.0.0-bridge.0",
```

Comment:
I believe with this jest update, and the removal of inject-loader, we can remove the babel-core dependency bridge.